src/pages/Login/Login.tsx

```diff
@@ -3,14 +3,14 @@ import React from 'react';
 import { useForm } from 'react-hook-form';
 
 import { LoginProps } from 'models/LoginProps';
-import { verifyUser } from 'service/verifyUser';
+import { verifyUser } from 'services/verifyUser';
```

Comment:
services must be more common (like reducers).
So it's must be `users` and not `verifyUser`, `logoutUser`, `checkUser`  e t c